src/check_https_domains.py

```diff
@@ -56,6 +56,10 @@ def parse_args():
         help='IP of the host. The address where the certificate will be '
              'retrieved from.'
     )
+    parser.add_argument(
+        '-p', dest='port', required=False, default=443, type=int,
```

Comment:
Instead of having `dest='port'`, we could also provide the possibility of `--port`. This will be then used as destination.
`required=False` is default. It is well known enough to not pass it explicitly.
Also the order of `type=int, default=443` makes more sense. First provide the type, then the default.
For all, no strong opinion. Up to you to change it or not.
```suggestion
        '-p', '--port', type=int, default=443,
```

Reply:
They make total sense. I'll change it :)